src/purchase_listinator/logic/purchase_list.clj

```diff
@@ -5,10 +5,11 @@
 
 (s/defn generate-new :- internal.purchase-list/PurchaseList
   [name :- s/Str]
-  (let [purchase-list {:id          (misc.general/squuid)
-                       :enabled     true
-                       :in-progress false}]
-    (assoc purchase-list :name name)))
+  {:id          (misc.general/squuid)
+   :enabled     true
+   :in-progress false
+   :status      :waiting
```

Comment:
What should be the initial `:status` for a new `PurchaseList`?

Reply:
I'm not sure yet :/

The Keyword :in-progress and :status were created for the same purpose, display to all the users that somebody is buying the items inside the list ([shopping cart in this image](https://github.com/gumberss/FinanceControlinatorDocs/blob/main/Features/Purchase%20Listinator/List%20View/v1/Lists%20View%20Tasks.md)).

It [was created](https://github.com/gumberss/FinanceControlinatorMobile/pull/66/files#diff-ebe0e414b857a375151d756a080c7b92f2b393f4f687175fc96f633ba45f1338R149) using the in-progress keyword, but I stopped this task once I noticed that I should create the shopping flow first instead and then decide how to do it, that's why [I moved this task to the shopping flow](https://github.com/gumberss/FinanceControlinatorDocs/blob/main/Features/Purchase%20Listinator/Shopping%20flow/v1/Init%20Purchase/Init%20purchase.md#client-side-3).
	
I don't like both of them inside the purchase-list entity, it is not of its bounded context. So far, this status [was moved to shopping](https://github.com/gumberss/PurchaseListinator/blob/main/src/purchase_listinator/models/internal/shopping.clj#L4) entity and what I'm thinking is that the :in-progress (in the purchase-list context) should be a memory property of the purchase-list view checking if there are some shopping in progress linked with the list.